rollup.config.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  input: './index.js',
+  output: {
+    file: 'dist/index.js',
```

Comment:
Can you please change the generated file name to "fast-freeze.js"

Reply:
Yes, sure!